react-delegationdashboard/src/components/LedgerTransactionStatus/txStatus.tsx

```diff
@@ -0,0 +1,9 @@
+const txStatus = {
+  pending: 'Pending',
+  notExecuted: 'Not Executed',
+  invalid: 'Invalid',
+  success: 'Success',
+  fail: 'fail',
```

Comment:
`fail: 'Fail',` as to match the rest of the statuses?

Reply:
Done